packages/browser/src/plugins/ajs-destination/index.ts

```diff
@@ -110,13 +118,19 @@ export class LegacyDestination implements Plugin {
       return
     }
 
-    this.integration = await loadIntegration(
-      ctx,
-      analyticsInstance,
-      this.name,
-      this.version,
+    const integrationSource =
+      this.integrationSource ??
+      (await loadIntegration(
```

Comment:
we might want to rename "loadIntegration" to something like "loadRemoteDestination" -- since load and build have overlapping meanings, and I could see this getting confusing.